examples/lighting-app/silabs/src/LightingManager.cpp

```diff
@@ -22,9 +22,14 @@
 #include "AppConfig.h"
 #include "AppTask.h"
 
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/ids/Attributes.h>
```

Comment:
include only the IDs that you need via `clusters/<cluster_name>/...`

You seem to need level control: https://github.com/project-chip/connectedhomeip/blob/master/zzz_generated/app-common/clusters/LevelControl/AttributeIds.h and such.

Reply:
Updated as per your comment.